mllib/src/main/scala/org/apache/spark/ml/clustering/GaussianMixture.scala

```diff
@@ -137,6 +137,13 @@ class GaussianMixtureModel private[ml] (
     sqlContext.createDataFrame(modelGaussians).toDF("mean", "cov")
   }
 
+  /**
+   * Returns a [[org.apache.spark.ml.util.MLWriter]] instance for this ML instance.
```

Comment:
I tested `@inheritdoc` tag to inherit ScalaDoc from parent and add additional notes. It works for Scala but it doesn't work in javaunidoc. So I guess we have to make those copies.